src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 auto;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__top {
+    display: inline-block;
+    width: 160px;
+    height: 134px;
+    padding: 0 3px;
+  }
+
+  &__imac {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__bottom {
+    display: flex;
+    margin-top: 40px;
+    flex-direction: column;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+    display: block;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__quality {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    display: block;
+    text-align: center;
+    color: #060b35;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__cost {
+    margin-top: 24px;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
+}
+
+.card__buy {
+  box-sizing: border-box;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  border-radius: 5px;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  color: #fff;
+  display: block;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  cursor: pointer;
+
+  &:hover {
+    color: #00acdc;
+    background: #fff;
+    border: 1px solid #00acdc;
+  }
+}
```

Comment:
The `.card__buy` selector is defined outside the `.card` block, which breaks the BEM convention. It should be nested within the `.card` block to maintain consistency with the rest of the component's styles.